CucumberTest/target/test-classes/features/Creat_User.feature

```diff
@@ -0,0 +1,12 @@
+Feature: Create User 

```

Comment:
noting from target folder goes to VCS
binaries should never never never be pushed to VCS